rhc-ose-ansible/roles/openshift-management/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+openshift_token:
+openshift_master_url: localhost
+openshift_login_insecure_flag: --insecure-skip-tls-verify=true
+openshift_login_insecure: False
+openshift_prune_builds_complete: 5
+openshift_prune_builds_failed: 1
+openshift_prune_builds_keep_younger: 1h0m0s
```

Comment:
This is the default value as per the [OpenShift Docs](https://docs.openshift.com/container-platform/3.3/admin_guide/pruning_resources.html#pruning-images). Can refine the default value if necessary. User has the ability to override the default value if desired